src/js/utils/MesosStateUtil.js

```diff
@@ -226,53 +226,7 @@ const MesosStateUtil = {
     });
   },
 
-  /**
-   * @param {{frameworks:array, completed_frameworks:array}} state
-   * @param {{id:string, executor_id:string, framework_id:string}} task
-   * @param {string} path
-   * @returns {string} task path
-   */
-  getTaskPath(state, task, path = "") {
-    const {
-      id: taskID,
-      framework_id: frameworkID,
-      executor_id: executorID
-    } = task;
-    const framework = MesosStateUtil.getFramework(state, frameworkID);
-    if (state == null || task == null || framework == null) {
-      return "";
-    }
-
-    let taskPath = "";
-    // Find matching executor or task to construct the task path
-    []
-      .concat(framework.executors, framework.completed_executors)
-      .every(function(executor) {
-        if (executor.id === executorID || executor.id === taskID) {
-          // Use the executor task path construct if it's a "pod" / TaskGroup
-          // executor (type: DEFAULT), otherwise fallback to the default
-          // app/framework behavior.
-          if (executor.type === ExecutorTypes.DEFAULT) {
-            // For a detail documentation on how to construct the task path
-            // please see: https://reviews.apache.org/r/52376/
-            taskPath = `${executor.directory}/tasks/${taskID}/${path}`;
-
-            return false;
-          }
-
-          // Simply use the executors directory for "apps" and "frameworks",
-          // as well as any other executor
-          taskPath = `${executor.directory}/${path}`;
-
-          return false;
-        }
-
-        return true;
-      });
-
-    return taskPath;
-  },
-
+  // TaskSystemLogsContainer
```

Comment:
Could you please explain me this comment, I do not get the context.

Reply:
Overlooked... `getTaskContainerID` is being used _only_ in `TaskSystemLogsContainer` that was a note for me